packages/@aws-cdk/aws-ec2/lib/security-group.ts

```diff
@@ -399,6 +403,8 @@ export class SecurityGroup extends SecurityGroupBase {
 
     this.allowAllOutbound = props.allowAllOutbound !== false;
 
+    this.disableInlineRules = props.disableInlineRules == true;
```

Comment:
Because this is ultimately JavaScript, we always use `===` for strict equality (the linter will probably yell at you if you don't).

If you can stomach it, in accepted JS style you can also use `!!props.disableInlineRules`.

Both are fine by me.